examples/terraform/druid-migration/README.md

```diff
@@ -0,0 +1,199 @@
+# Launch Druid on Compute Engine using Terraform
+
+Using this code repository users can launch there  Druid cluster on GCP using terraform.
+
+## Resources to launch
+
+* CloudSQL Instance
+ 
+* Dataproc Cluster
+ 
+* Zookeeper Cluster
+    
+* Kafka Cluster
+    
+* Druid Cluster
+
+
+## Architectural diagram
+![Druid Architecture](https://storage.googleapis.com/pso-druid-migration/Druidarchitecture.png "Logo Title Text 1")
+
+## setup the environment
+
+1. Clone the repository.
+```
+git clone <repo_link>
+cd druid_terraform/
+```
+2. [Create a service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating_a_service_account) and grant the following roles to the service account by following these [instructions](https://cloud.google.com/iam/docs/granting-roles-to-service-accounts#granting_access_to_a_service_account_for_a_resource)
+
+    * Service networking admin
+ 
+    *  iam.serviceAccountUser
+ 
+    * roles/compute.admin
+    
+    * roles/compute.securityAdmin
+    
+    * roles/storage.admin
+    
+    * roles/cloudsql.admin
+ 
+    * roles/dataproc.editor
+    
+	
+## Input variables
+
+Below are the variables which we are using while deploying the infrastructure using terraform.
```

Comment:
I suggest autogenerating this using terraform-docs.

Reply:
Generated and committed changes on my forked repo